pkgdb2/api/packages.py

```diff
@@ -781,7 +806,9 @@ def api_package_info(pkgname=None):
 @API.route('/packages')
 @API.route('/packages/<pattern>/')
 @API.route('/packages/<pattern>')
-def api_package_list(pattern=None):
+@API.route('/package/<namespace>/<pattern>/')
+@API.route('/package/<namespace>/<pattern>')
+def api_package_list(namespace=None, pattern=None):
```

Comment:
Should the default value for ``namespace`` be ``'rpms'`` here in the function signature?  That would maintain symmetry with the ``api_package_info`` function above.